guides/release/testing/testing-components.md

```diff
@@ -103,31 +121,33 @@ module('Integration | Component | pretty-color', function(hooks) {
     // set the outer context to red
     this.set('colorValue', 'red');
```

Comment:
Again, should these still use set?

https://rfcs.emberjs.com/id/0785-remove-set-get-in-tests